### PR TITLE
remove Secrets permission from user access list

### PR DESCRIPTION
**Pull request by ashwindasr**
`Contributor` and `Maintainer` does not have any permissions for `Secrets`, ref docs [here](https://github.com/konflux-ci/architecture/blob/main/ADR/0011-roles-and-permissions.md).

Bug found by @thegreyd

### CODE REVIEW

File: src/components/UserAccess/UserAccessForm/PermissionsTable.tsx [hunk 1]
```diff
@@ -31,7 +31,6 @@ const permissions: Record<WorkspaceRole, Record<string, Permission[]>> = {
     'Remote Secrets': [Permission.Read],
```
[the remark this answers is not in the document]
Comment (ashwindasr) [reply src/components/UserAccess/UserAccessForm/PermissionsTable.tsx]: Thanks @sahil143, updated PR

[the remark this answers is not in the document]
Comment (ashwindasr) [reply src/components/UserAccess/UserAccessForm/PermissionsTable.tsx]: But do we need to remove it from here as well? https://github.com/konflux-ci/architecture/blob/main/ADR/0011-roles-and-permissions.md

cc @arewm

[the remark this answers is not in the document]
Comment (arewm) [reply src/components/UserAccess/UserAccessForm/PermissionsTable.tsx]: I am hesitant to update the ADR as I think we actually need a "bigger" ADR about deprecating remote secrets.

**Comment by sahil143** [comment]
> Is this still a valid place for a PR or should it instead be created against https://github.com/konflux-ci/konflux-ui?

Yes, this is a small change and can be merged in hac-dev.

**Review by sahil143 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**